Header header 

uint8[] stability

float32[] x_cur
float32[] z_cur
float32[] radius_cur
float32[] height_cur

float32[] x_old
float32[] z_old
float32[] radius_old
float32[] height_old